jbmc/unit/java_bytecode/java_object_factory/gen_nondet_string_init.cpp

```diff
@@ -46,7 +46,7 @@ SCENARIO(
     symbol_typet java_string_type("java::java.lang.String");
     symbol_exprt expr("arg", java_string_type);
 
-    object_factory_parameterst object_factory_parameters;
+    java_object_factory_parameterst object_factory_parameters;
```

Comment:
Should be merged into the earlier commit introducing `java_object_factory_parameterst`.